src/main/flight/pid.c

```diff
@@ -585,6 +635,12 @@ rollAndPitchTrims_t *angleTrim, rxConfig_t *rxConfig)
         }
 
         PTerm -= gyroADCQuant * dynP8[axis] * 0.003f;
+
+        // Pterm low pass
+        if (pidProfile->pterm_cut_hz) {
+            PTerm = filter_pt1(PTerm, &PTerm_state[axis], pidProfile->pterm_cut_hz);
```

Comment:
Not sure if I am getting this one? 
PTerm_state is the old one and PTerm the new one and is same as it was before. I will make PTermState of the new one and separate PR should be than made for normalizing it later?